test/test_monitor.py

```diff
@@ -129,6 +131,10 @@ def tearDown(self):
         for temp_dir in self.temp_dirs:
             temp_dir.cleanup()
 
+    @unittest.skipIf(
+        sys.version_info >= (3, 13),
+        "numpy failure, likely caused by old tensorboard version",
+    )
```

Comment:
Do we want to create issue for the code owners of this?

Reply:
Will do